code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -134,7 +137,7 @@ pub mod pallet {
 			/// FNFT Instance Id
 			fnft_instance_id: T::FinancialNftInstanceId,
 			/// Reward multiplier
-			reward_multiplier: Perbill,
+			reward_multiplier: FixedU64,
```

Comment:
Would it be worth making this type generic?

Reply:
Made a ticket, we can review it at a later date: https://app.clickup.com/t/34540jh